### DISCUSSION TITLE
minio automating policy settings

### DISCUSSION THREAD

**Discussion by rbs-cloud** [minio/minio#13594] [Q&A] [answered]
Hi Team,
we are able to set the policy through mc like below

mc alias set minio-user http://LB-IP:9000 admin123 admin123
mc admin policy set minio-user readwrite user='uid=miniouser(ldap username),ou=People,dc=ldapdcname,dc=com'

if successfully done above steps ,then we are able to sent the sts requests successfully and got the tempororaly credential's but we wanted to do same steps through  program or some automated way so that policy should apply without mc command

Could you please guide me on this

**Comment by harshavardhana** [comment, marked as answer]
> if successfully done above steps ,then we are able to sent the sts requests successfully and got the tempororaly credential's but we wanted to do same steps through program or some automated way so that policy should apply without mc command

Automation is use `mc` command in your deployment spec.

**Reply by rbs-cloud** [reply]
Hi @harshavardhana 

Automation is use mc command in your deployment spec.----- could you please share snippet code
